src/nullspace.jl

```diff
@@ -147,11 +147,37 @@ function nullspace(::Order{2}, M::ITensor, left_inds, right_inds; tags="n", kwar
   return Ñ
 end
 
-function nullspace(T::ITensor, is...; tags="n", kwargs...)
+"""
+    nullspace(T::ITensor, left_inds...; tags="n", atol=1E-12, kwargs...)
+
+Viewing the ITensor `T` as a matrix with the provided `left_inds` viewed
+as the row space and remaining indices viewed as the right indices or column space,
+the `nullspace` function computes the right null space. That is, it will return
+a tensor `N` acting on the right indices of `T` such that `T*N` is zero.
+The returned tensor `N` will also have a new index with the label "n" which
+indexes through the 'vectors' in the null space.
+
```

Comment:
Should we add a warning that the implementation might change in the future and therefore the tolerance behavior might change?

Reply:
That's a good idea – will add it

Reply:
Also maybe warn that for input tensors with very small singular values the size of the returned nullspace could be very sensitive to the tolerance? Maybe that is already inherent to any floating implementation of a nullspace function.